StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1009UnitTests.cs

```diff
@@ -418,6 +418,29 @@ public Foo(int i) : base()
             await this.TestWhitespaceInStatementOrDeclAsync(invalidStatement, validStatement, expected).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// This is a regression test for DotNetAnalyzers/StyleCopAnalyzers#1064 "SA1009: mis-fires for verbatim
+        /// strings"
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task TestVerbatimStringArgumentAsync()
+        {
+            string testCode = @"using System;
+public class Foo
+{
+    public void Method()
+    {
+        Console.WriteLine(@""
+    Written content...
+"");
```

Comment:
It would still just be a verbatim string. I looked into it a bit and it doesn't seem like a second test with indentation would improve the overall quality of the regression test.